src/app/inventory/store/d2-item-factory.ts

```diff
@@ -269,17 +276,29 @@ export function makeItem(
 
   const itemType = normalBucket.type || 'Unknown';
 
-  const dmgName =
-    damageTypeNames[
-      (instanceDef ? instanceDef.damageType : itemDef.defaultDamageType) || DamageType.None
-    ];
-
   // https://github.com/Bungie-net/api/issues/134, class items had a primary stat
   const primaryStat =
     (itemDef.stats && itemDef.stats.disablePrimaryStatDisplay) || itemType === 'Class'
       ? null
       : (instanceDef && instanceDef.primaryStat) || null;
 
+  const energyInfo =
```

Comment:
Can we just use the Destiny energy info as-is, rather than translating it into a similar shape of our own?